src/Annotation/Tags.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Dealroadshow\K8S\Framework\Annotation;
+
+use Attribute;
+
+#[Attribute(Attribute::TARGET_CLASS)]
+class Tags
+{
+    /**
+     * @var string[]
+     */
+    private array $tags;
+
+    public function __construct(string ...$tags)
+    {
+        $this->tags = $tags;
+    }
+
+    /**
+     * @return string[]
+     */
+    public function getTags(): array
```

Comment:
I think that class is already named Tags, so we can name this method `all()` or `get()`